apps/main/src/loan/components/PageLlamaMarkets/Page.tsx

```diff
@@ -67,12 +68,13 @@ export const LlamaMarketsPage = (props: CrvUsdServerData) => {
   const { address } = useAccount()
   const { data, isError, isLoading } = useLlamaMarkets(address)
   const minLiquidity = useUserProfileStore((s) => s.hideSmallPools) ? SMALL_POOL_TVL : 0
+  const isTiny = useMediaQuery('(max-width:400px)')
```

Comment:
This would be great to have in in aforementioned hook as well 😄

Reply:
Hmm not sure, this is very specific to this component :thinking: